Jenkinsfile

```diff
@@ -4,8 +4,7 @@ library 'pipeline-library'
 def canPublish = env.BRANCH_NAME == '1_X'
 
 buildNPMPackage {
-  labels = 'osx && git && npm-publish && xcode'
-  downstream = ['../appc-cli']
```

Comment:
node-ios-device isn't bundled into appc-cli anymore so removing this to avoid unnecessary builds